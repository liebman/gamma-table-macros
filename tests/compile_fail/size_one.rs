use gamma_table_macros::gamma_table;

// This should fail to compile because size 1 is less than the minimum of 3
gamma_table! {
    name: INVALID_SIZE_ONE_TABLE,
    entry_type: u8,
    gamma: 2.2,
    size: 1
}

fn main() {} 