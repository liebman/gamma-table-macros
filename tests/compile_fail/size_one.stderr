error: Size must be at least 3 to create a meaningful gamma table. Smaller sizes only have min and max values.
 --> tests/compile_fail/size_one.rs:5:11
  |
5 |     name: INVALID_SIZE_ONE_TABLE,
  |           ^^^^^^^^^^^^^^^^^^^^^^
